admin-dev/themes/default/sass/partials/_buttons.sass

```diff
@@ -0,0 +1,10 @@
+.btn
```

Comment:
Not the moment to move from SASS to SCSS ?

Reply:
You finally got me...

I'm gonna do it in another PR, when PR touching default will be merged